docs/admin/kubernetes-ops/cert-management.md

```diff
@@ -3,16 +3,203 @@ tags:
   - administration
   - kubernetes
   - operations
+  - certificate management
+  - certification and key rotation
 ---
 
-## This content is in progress
+## Overview
 
-The Janssen Project documentation is currently in development. Topic pages are being created in order of broadest relevance, and this page is coming in the near future.
+Rotating Certificates and Keys in Kubernetes setup
 
-## Have questions in the meantime?
+!!! Note
+    `janssen-config-cm` in all examples refer to jans installation configuration parameters where `janssen` is the `helm-release-name`.
+       
+    
+## Web (Ingress)
+        
+| Associated certificates and keys |
+| -------------------------------- |
+| /etc/certs/web_https.crt         |
+| /etc/certs/web_https.key         |
+        
+### Rotate
+        
+1.  Create a file named `web-key-rotation.yaml` with the following contents :
+        
+    ```yaml
+    apiVersion: batch/v1
+    kind: Job
+    metadata:
+      name: web-key-rotation
+    spec:
+      template:
+        metadata:
+          annotations:
+            sidecar.istio.io/inject: "false"                  
+        spec:
+          restartPolicy: Never
+          containers:
+            - name: web-key-rotation
+              image: janssenproject/certmanager:1.0.5
```

Comment:
check the images. I meant to use version `1.0.5`. Latest version there is `1.0.5-1`.